packages/thrift-client/src/main/connections/axios.ts

```diff
@@ -19,10 +19,19 @@ export class AxiosConnection<TClient> extends HttpConnection<TClient> {
     this.request = requestApi
     this.request.defaults.responseType = 'arraybuffer'
     this.request.defaults.baseURL = `http://${this.hostName}:${this.port}`
+    this.request.defaults.headers = {
+      'host': this.hostName,
+      'connection': 'keep-alive',
```

Comment:
I just tested the headers from the existing Axios request:
```
  "accept": "application/json, text/plain, */*",
  "content-type": "application/x-www-form-urlencoded",
  "user-agent": "axios/0.16.2",
  "content-length": "87",
  "host": "localhost:8080",
  "connection": "close"
```
If you change the connection to keep-alive, does anything need to be set differently on the connection to support it? Setting keep-alive when the connection actually closes may add a bit of overhead to the server.

Reply:
Yeah, good point. The only reason I did this is because the apache implementation did this. Will remove.

Reply:
I took a look at keep-alive for Axios. It may have performance benefits if one service calls another multiple times. It does look like it needs to be configured.
```
  // `httpAgent` and `httpsAgent` define a custom agent to be used when performing http
  // and https requests, respectively, in node.js. This allows options to be added like
  // `keepAlive` that are not enabled by default.
  httpAgent: new http.Agent({ keepAlive: true }),
  httpsAgent: new https.Agent({ keepAlive: true }),
```

See: https://github.com/axios/axios

Maybe a separate PR.